versions/3.1.1.md

```diff
@@ -2810,7 +2818,12 @@ components:
             type: boolean
 ```
 
-a payload like this:
+The `MyResponseType` schema will use the discriminator defined by the `Pet`
+schema because it is part of the `Cat`, `Dog`, and `Lizard` schemas in the
+`oneOf`. The behavior if not all schemas define a `discriminator` and they are
+not all the same is undefined.
```

Comment:
This would need to be removed if the `oneOf` addition to the example is removed.